Makefile

```diff
@@ -19,7 +19,7 @@ lint:
 
 .PHONY: test
 test:
-	pytest --cov=pydantic
```

Comment:
remove these changes, the problem should be fixed by Frozenball/pytest-sugar#133 which will hopefully be merged soon.

If the problems persist I'll remove `pytest-sugar` completely.